lib/LaTeXML/Package/physics.sty.ltxml

```diff
@@ -96,43 +100,10 @@ sub phys_readArg {
       if $required; }
   return (wantarray ? ($arg, $open, $close) : $arg); }
 
-#----------------------------------------------------------------------
-# Construction aids
-
-# Note: This could be applicative-power ???
-sub phys_applyPower {
-  my ($cfunc, $pfunc, $power) = @_;
-  my ($cargs, $pargs) = dualize_arglist('#1', $power);
-  if ($cfunc) {
-    $cfunc = Invocation(T_CS('\@APPLY'),
-      Tokens(Invocation(T_CS('\@CSYMBOL'), T_OTHER('power')),
-        $cfunc, $$cargs[0])); }
-  if ($pfunc) {
-    $pfunc = Tokens($pfunc, Invocation(T_CS('\@@POSTSUPERSCRIPT'), $$pargs[0])); }
-  return ($cfunc, $pfunc); }
-
-# TODO: Work out the optimum assumptions for missing $cfunc, $pfunc, @args...
-sub phys_applyFunction {
-  my ($cfunc, $pfunc, $size, $open, $close, @args) = @_;
-  $cfunc = undef unless $cfunc && $cfunc->unlist;
-  $pfunc = undef unless $pfunc && $pfunc->unlist;
-  my ($cargs, $pargs) = dualize_arglist('#1#2#3#4#5#6#7#8#9', @args);
-  if (!$cfunc && !@args) {    # No need for dual.
-    return Invocation(T_CS('\@WRAP'),
-      Tokens(($pfunc ? ($pfunc, T_CS('\@APPLYFUNCTION')) : ()),
-        phys_open($size, $open), phys_close($size, $close))); }
-  else {
-    return Invocation(T_CS('\DUAL'), undef,
-      ($cfunc && @args
-        ? Invocation(T_CS('\@APPLY'), Tokens($cfunc, @$cargs))
-        : ($cfunc
-          ? $cfunc
-          : @$cargs)),        # Hopefully only one ???
-      Invocation(T_CS('\@WRAP'),
-        Tokens(
-          ($pfunc ? ($pfunc, T_CS('\@APPLYFUNCTION')) : ()),
-          phys_open($size, $open), JoinTokens(T_OTHER(','), @$pargs), phys_close($size, $close))));
-} }
+sub phys_revArg {
```

Comment:
I did a quick search and all calls to `phys_revArg` take a constant number literal and wrap it as `phys_revArg(T_XMArg(1))`. Why not call the function directly on a number literal and save the extra symbols, as in `phys_revArg(1)` checking if `ref $arg` is set and wrapping with `T_XMArg` if not?